src/main/java/com/onarandombox/MultiverseCore/commands/CreateCommand.java

```diff
@@ -46,10 +46,15 @@ public CreateCommand(MultiverseCore plugin) {
         this.addCommandExample("/mv create " + ChatColor.GOLD + "moonworld" + ChatColor.GREEN + " normal" + ChatColor.DARK_AQUA + " -g BukkitFullOfMoon");
         this.worldManager = this.plugin.getMVWorldManager();
     }
-
+	
+	private String trimWorldName(String userInput) {
+        // Removes relative paths.
+        return userInput.replaceAll("^[./\\\\]+", "");
+    }
```

Comment:
Why wouldn't `\\` suffice (in place of `\\\\`)?

Also, please use spaces for indenting.

Reply:
I copied the function present in 
https://github.com/Multiverse/Multiverse-Core/blob/master/src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java#L99

Reply:
I figured out why `\\\\` is needed.

The first `\\` is to escape the regex escape char (`\`). The second `\\` is to escape the backslash.